src/main/java/bio/terra/app/controller/AdminApiController.java

```diff
@@ -61,4 +77,53 @@ public ResponseEntity<JobModel> registerDrsAliases(List<DrsAliasModel> aliases)
     String jobId = drsService.registerDrsAliases(aliases, userReq);
     return ControllerUtils.jobToResponse(jobService.retrieveJob(jobId, userReq));
   }
+
+  @Override
+  public ResponseEntity<DatasetModel> adminRetrieveDataset(UUID id) {
+    List<DatasetRequestAccessIncludeModel> include =
+        List.of(
+            DatasetRequestAccessIncludeModel.ACCESS_INFORMATION,
```

Comment:
> Summary endpoints don't return any protected data, right? I know it is a requirement to not return any protected data, and it was suggested to use the summary endpoints for this purpose, but I just wanted to double check.

If we want to only return summary information, then we need to leave off "access_information" specifically for the azure case. We provide signed urls for the azure case so anyone retrieving this information would have direct access to the source data. We do not do the same for the gcp case: We provide information for how to see data in BigQuery, but no direct access is given.

Reply:
This also applies to snapshots.

Reply:
thanks!